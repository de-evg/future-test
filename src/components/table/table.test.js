import React, {createElement} from 'react';
import renderer from "react-test-renderer";
import Table from "./table";

it(`Should Table render correctly`, () => {
  const tableBody = createElement(`tbody`);
  const tableHead = createElement(`thead`);
  const tree = renderer
    .create(
      <Table
        TableBody={tableBody}
        TableHead={tableHead}
      />
    )
    .toJSON();

    expect(tree).toMatchSnapshot();
});