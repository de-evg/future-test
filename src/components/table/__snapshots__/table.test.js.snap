// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Table render correctly 1`] = `
<table
  className="table"
>
  <thead />
  <tbody />
</table>
`;
